app/src/test/java/com/github/sdpteam15/polyevents/model/ZoneTest.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.sdpteam15.polyevents.model
+
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+
+class ZoneTest {
+    lateinit var zone: Zone
+
+    val zoneName = "zone A"
+    val location = "Esplanade"
```

Comment:
How do you see the colation? Because I had in mind that it would be a list of Latitude,Longitude defining the boundaries of the zone

Reply:
It's a placeholder for now. We can update it later with you see fit for the Map functionality.

Reply:
This goes the same for all entities.